terra/VteObject.py

```diff
@@ -55,15 +54,22 @@
     "(?:news:|man:|info:)[[:alnum:]\\Q^_{|}~!\"#$%&'()*+,./;:=?`\\E]+"]
 
 class VteObjectContainer(Gtk.HBox):
-    def __init__(self, parent, bare=False, progname=ConfigManager.get_conf('general', 'start_shell_program'), pwd=None):
+    def __init__(self, parent, bare=False, progname=None, pwd=None):
```

Comment:
Why do you prefer to remove the default parameter and check if progname is set later?
Is it about readability?